lib/assigner.rb

```diff
@@ -250,13 +250,19 @@ def assign(assign_to, silent: false)
       end
     end
     if !silent
+      custom_fields = { "action_code_who" => assign_to.is_a?(User) ? assign_to.username : assign_to.name }
+
+      if post_target?
+        custom_fields.merge!("action_code_href" => "/p/#{@target.id}")
```

Comment:
I don't think this will work with subfolder installs since we aren't using `getURL`. Should the core PR be updated to support them?

Reply:
Great catch! thank you, I adjusted core PR:
`https://github.com/discourse/discourse/pull/14816/commits/b2f5bd2c6863283b25371815f3e1b18bdddae1fb`